lighthouse-core/runner.js

```diff
@@ -93,9 +93,10 @@ class Runner {
       const resultsById = {};
       for (const audit of runResults.auditResults) resultsById[audit.name] = audit;
 
-      let report;
+      let cats;
```

Comment:
![cats](https://media.giphy.com/media/kmFNdsZfgMo7e/giphy.gif)

can we go `categories` though :)